src/main/java/software/amazon/cloudwatchlogs/emf/logger/MetricsLogger.java

```diff
@@ -189,20 +192,56 @@ public MetricsLogger resetDimensions(boolean useDefault) {
      * @param key is the name of the metric
      * @param value is the value of the metric
      * @param unit is the unit of the metric value
+     * @param storageResolution is the resolution of the metric
```

Comment:
Can we put a link to the CW documentation regarding the High Resolution in the @param description ? As this is public facing method 

Url : https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/publishingMetrics.html#high-resolution-metrics

Reply:
Thanks for the suggestion. Updated!